ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: triage
```

Comment:
Ditto, and maybe add an `enhancement` label? Or do not add any labels at all, they will be added manually by developers after issue consideration.